leetcode/string.md

```diff
@@ -0,0 +1,22 @@
+# Intervals
```

Comment:
# Invalid PULL Commits

** Invalid Commit Message: LEETCODE 1022 Refactored reverse-words-in-a-string **
! Message must contain group number in ['1021', '1022']
! Message must start with ['Added', 'Deleted', 'Fixed', 'Refactored', 'Moved']

** Invalid Commit Message: Merge remote-tracking branch 'origin/reverse-words-in-a-string-iii' into reverse-words-in-a-string-iii

# Conflicts:
#	leetcode/string.md **
! Message must start with code word in ['LEETCODE', 'GENERATOR', 'HEXNUMBER', 'TRIANGLE', 'ITERATOR', 'REQUESTS']
! Message must contain group number in ['1021', '1022']
! Message must start with ['Added', 'Deleted', 'Fixed', 'Refactored', 'Moved']